tests/x11regressions/gnomecase/application_starts_on_login.pm

```diff
@@ -0,0 +1,117 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
```

Comment:
As this is a new file you are adding please just state the copyright years for the time you worked on this file, I guess only 2016, so it should say
```
# SUSE's openQA tests
#
# Copyright © 2016 SUSE LLC
```